Reneiski_Homework1/Reneiski_Homework1/Lecture7/Task1/Potato.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace Reneiski_Homework1.Lecture7.Task1
+{
+    public abstract class Potato
+    {
+        public string? sortOfPotato;
+
+        public abstract void WayOfCooking();
+
+        public abstract string CheckTheSort { get; set; }
+
+        public void BadPotato()
```

Comment:
please change name of method so that it illustrate what it is doing

Reply:
Fixed